Framework/PythonInterface/plugins/algorithms/InelasticEMUauReduction.py

```
@@ -6,7 +6,7 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 import os
 
-# import debugpy
+import debugpy
```

Ruff (0.12.2)

Import for `debugpy` found

(T100)